src/meta/buying_options.yaml

```
@@ -1,18 +1,82 @@
-carrot:
+golden_carrot:
   type: item
   name: Golden Carrots
   item: minecraft:golden_carrot
   price: 200
   amount: 8
+torch:
+  type: item
+  name: Torches
+  item: minecraft:torch
+  price: 100
+  amount: 32
+hopper:
+  type: item
+  name: Hoppers
+  item: minecraft:hopper
+  price: 500
+  amount: 16
+cobblestone:
+  type: item
+  name: Cobblestone
+  item: minecraft:cobblestone
+  price: 10
+  amount: 64
+book:
+  type: item
+  name: Books
+  item: minecraft:book
+  price: 200
+  amount: 4
+firework:
+  type: item
+  name: Fireworks
+  item: minecraft:firework_rocket{Fireworks:{Flight:2}}
+  price: 100
+  amount: 32
```

```suggestion
  price: 500
  amount: 32
```
Is this for using with elytras?

Either way, too cheap.